FearlessUtils/Classes/Crypto/BIP32JunctionFactory.swift

```diff
@@ -0,0 +1,92 @@
+import Foundation
+import BigInt
+import IrohaCrypto
+
+public struct BIP32JunctionFactory: JunctionFactoryProtocol {
```

Comment:
We have duplicated logic regarding junction parsing compared with JunctionFactory. Consider to turn to the class and make base class with common logic.

Reply:
Done